Wave/Wave/SectionType.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SectionType.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 6/27/16.
+//  Copyright © 2016 Blue Line Labs. All rights reserved.
```

Comment:
Our name is BlueLine Labs. Should prob update this in all your copyrights (and Xcode)